src/App.tsx

```diff
@@ -1,173 +1,52 @@
-import { Loader } from './components/Loader';
-
+import {
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
 import './App.scss';
+import { HomePage } from './components/HomePage/HomePage';
+import { PeoplePage } from './components/PeoplePage/PeoplePage';
+import { NavBar } from './components/NavBar/NavBar';
+
+export const App = () => {
+  return (
+    <div data-cy="app">
+      <NavBar />
+
+      <main className="section">
+        <div className="container">
```

Comment:
if I decided to add a new page which will not include a `NavBar`, how can I do this?
I suggest you create a `Layout` component and wrap all your routes in the `Layout` route.
```
<Routes>
       <Route path="/" element={<Layout/>}>
             ...
        </Route>
</Routes>
```
In Layout, you can use `Outlet` from `react-router-dom` to render your components inside.